python-wrapper/src/neo4j_viz/gds.py

```diff
@@ -55,18 +57,20 @@ def _fetch_rel_dfs(gds: GraphDataScience, G: Graph) -> list[pd.DataFrame]:
 def from_gds(
     gds: GraphDataScience,
     G: Graph,
-    size_property: Optional[str] = None,
-    additional_node_properties: Optional[list[str]] = None,
-    additional_db_node_properties: Optional[list[str]] = None,
-    node_radius_min_max: Optional[tuple[float, float]] = (3, 60),
+    node_properties: Optional[list[str]] = None,
+    db_node_properties: Optional[list[str]] = None,
     max_node_count: int = 10_000,
 ) -> VisualizationGraph:
     """
     Create a VisualizationGraph from a GraphDataScience object and a Graph object.
 
-    All `additional_node_properties` will be included in the visualization graph.
-    If the properties are named as the fields of the `Node` class, they will be included as top level fields of the
-    created `Node` objects. Otherwise, they will be included in the `properties` dictionary.
+    By default:
+
+    * the caption of a node will be based on its `labels` property.
+    * the caption of a relationship will be based on its `relationshipType` property.
```

Comment:
I wonder if it's confusing to talk about properties here since they're not actually properties in the GDS sense